interface/main/mobile/camera.php

```diff
@@ -0,0 +1,617 @@
+ *
+ * @package OpenEMR
+ * @author Ray Magauran <magauran@MedExBank.com>
+ * @link http://www.open-emr.org
+ * @copyright Copyright (c) 2018 MedEx <magauran@MedExBank.com>
+ * @license https://www.gnu.org/licenses/agpl-3.0.en.html GNU Affero General Public License 3
+ */
+require_once "../../globals.php";
+require_once "$srcdir/patient.inc";
+require_once "$srcdir/options.inc.php";
+require_once $GLOBALS['srcdir']."/../vendor/mobiledetect/mobiledetectlib/Mobile_Detect.php";
+require_once "m_functions.php";
+
+$detect             = new Mobile_Detect;
+$device_type        = ($detect->isMobile() ? ($detect->isTablet() ? 'tablet' : 'phone') : 'computer');
+$script_version     = $detect->getScriptVersion();
+
+$uspfx              = substr(__FILE__, strlen($webserver_root)) . '.';
+$setting_mFind      = prevSetting('', 'setting_mFind', 'setting_mFind', 'byRoom');
+$setting_mRoom      = prevSetting('', 'setting_mRoom', 'setting_mRoom', '');
+$setting_mCategory  = prevSetting('', 'setting_mCategories', 'setting_mCategories', '');
+
+if (($_POST['setting_new_window']) ||
+    ($_POST['setting_mFind'])) {
+    exit();
+}
+
+if (!empty($_GET['desktop'])) {
+    $_SESSION['desktop'] = $_GET['desktop'];
+} else {
+    $_SESSION['desktop'] = "";
+}
+$categories         = array();
+$doc                = array();
+$display            = "photo";
+$pid                = '';
+
+// If “Go to full website” link is clicked, redirect mobile user to main website
+if (!empty($_SESSION['desktop'])) {
+    $desktop_url = $GLOBALS['webroot']."/interface/main/tabs/main.php";
+    header("Location:" . $desktop_url);
+}
+
+/**
+ * We have a preference to take the photo of the person in a specific room
+ * Who is in that room?  Is there more than one 'cause patient_tracker is not up-to-date?
+ * We need the fname,lname and pid(s) for person(s) in Room X.
+ * Let's get those data points now.
+ */
+if (($setting_mFind == 'byRoom') && (!empty($setting_mRoom))) {
+    $query = "select fname,lname,pid from patient_data
+              where pid in (
+                SELECT pc_pid FROM `openemr_postcalendar_events`
+                where pc_room=? and pc_apptstatus in (
+                  SELECT option_id FROM list_options
+                  WHERE list_id = 'apptstat' and toggle_setting_1 ='1' and activity='1'
+                ) and
+                pc_eventDate = CURDATE() )";
+    $results_byRoom = sqlStatement($query, array($setting_mRoom));
+}
+
+?><!doctype html>
+<html style="cursor: pointer;">
+<?php
+    common_head();
+?>
+
+<script>
+    var projects = [
+        {
+            label: "<?php echo xla('Select Document Category'); ?>"
+        }<?php
+        $categories =  sqlStatement("Select * from categories");
+        
+        while ($row1 = sqlFetchArray($categories)) {
+            echo ',
+                {
+                    label: "'.attr($row1['name']).'",
+                    catID: "'.attr($row1['id']).'"
```

Comment:
```
label: '.js_escape($row1['name']).',
catID: '.js_escape($row1['id']).'
```
This is the pure js escaping function when not translating. Note I removed the quotes.